src/python/deepgnn/logging_utils.py

```diff
@@ -126,6 +126,22 @@ def setup_default_logging_config(enable_telemetry: bool = False):
     _init = True
 
 
+def set_logger(customized_logger: logging.Logger) -> None:
```

Comment:
@DoWhatILove Could you elaborate why do we need a setlogger function instead of adding custom handlers to the existing logger? This seems to be a standard practice:
https://docs.python.org/3/howto/logging-cookbook.html#using-logging-in-multiple-modules
Standard library also doesn't have any setters:
https://docs.python.org/3/library/logging.html#module-level-functions